src/averageAges.js

```diff
@@ -54,8 +66,22 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+function hasMother(crowd, humanMother) {
```

Comment:
Consider renaming the `crowd` parameter to `people` and `humanMother` to `motherName` for better readability.